code/_onclick/hud/fullscreen.dm

```diff
@@ -1,4 +1,6 @@
-/mob/proc/overlay_fullscreen(category, type, severity)
+/mob/proc/overlay_fullscreen(category, type, severity, overlay_alpha)
+	if(!overlay_alpha)
+		overlay_alpha = 255
```

Comment:
Is there a problem if we just assign a default arg value?
```suggestion
/mob/proc/overlay_fullscreen(category, type, severity, overlay_alpha = 255)
```

Reply:
no problem, i'll change it